fluentd/elasticsearch.yml

```diff
@@ -0,0 +1,35 @@
+stack: kontena/fluentd-elasticsearch
+version: 0.1.0
+description: Fluent forwarding to Elasticsearch
+variables:
+  host:
+    type: string
+    default: elasticsearch.${GRID}.kontena.local
+    from:
+      prompt: Elasticsearch host
+  user:
+    type: string
+    default: elastic
+    from:
+      prompt: Elasticsearch user
+
+services:
+  agent:
+    build:
```

Comment:
Do we need this?

Reply:
No, had this during testing. Will remove.